openverse_catalog/dags/common/loader/paths.py

```diff
@@ -5,7 +5,7 @@
 from typing import Optional
 
 from airflow.models import TaskInstance
-from util.loader.ingestion_column import _fix_ingestion_column
+from common.loader.ingestion_column import _fix_ingestion_column
```

Comment:
Not related to this PR: I don't know if you've already noticed, but the `_fix_ingestion_column` can be retired :) It was used to convert old format into new, but we do not have that old format of tsvs anywhere anymore.

Reply:
Great to know! I'll make a new issue for that 😄 I love that you keep pointing these out, I'm happy to take stuff out we no longer need to carry forward 💪